src/lib.rs

```diff
@@ -19,6 +19,9 @@ extern crate packed_struct_codegen;
 pub mod alloc;
 pub mod gc;
 
+// Re-export public APIs
```

Comment:
I think the comment is redundant and can be removed.

Reply:
Fixed: 1da75a6e56d35720ebc243bc60d3a89bb63e7704